src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,37 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        ManufacturerService manufacturerService = (ManufacturerService) injector
+                .getInstance(ManufacturerService.class);
+        Manufacturer manufacturerToyota = new Manufacturer("Toyota", "Japan");
+        Manufacturer manufacturerFord = new Manufacturer("Ford", "United States");
+        manufacturerService.create(manufacturerToyota);
+        manufacturerService.create(manufacturerFord);
+        System.out.println(manufacturerService.get(1L));
+        manufacturerService.getAll().forEach(System.out::println);
+        Manufacturer manufacturerFerrari = new Manufacturer(1L, "Ferrari", "Italy");
+        System.out.println(manufacturerService.update(manufacturerFerrari));
+        System.out.println(manufacturerService.delete(2L));
```

Comment:
It's better to separate logically grouped lines of code by empty blocks for the sake of readability and easier understanding either in the future or by other devs.